pallets/dactr/src/benchmarking.rs

```diff
@@ -204,6 +204,36 @@ benchmarks! {
 		let _data_root =submitted_data::extrinsics_root::<T::SubmittedDataExtractor, _>(once(&opaque));
 	}
 
+	// This benchmark is not directly used by extrinsic.
+	// It is mostly used to check that the weight is lower or approximately equal the `data_root` benchmark
+	data_root_batch {
+		let i in 0..(2 * 1024 * 1024);
```

Comment:
`i` should be greater than `max_tx_size`,  and we should use the `max_block_len` (not sure about this name)
```suggestion
		let i in max_tx_size..max_block_len;
```

It would be nice if the step is `max_tx_size` too

Reply:
Here, i go from 0 to 2mb et make n transactions from it, so it will benchmark the creation of dataroot until the maximum that we can put in a block.

Why should we go from max_tx_size to max_block_len which is basically the same from 512kb to 2mb ?